conda-recipes/meta.yaml

```diff
@@ -27,22 +27,33 @@ requirements:
     - vtk=8.2
     - eigen
     - tbb-devel=2020
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
```

Comment:
These are redundant. Proposing to remove them in https://github.com/CompuCell3D/CompuCell3D/pull/509.